airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -100,6 +100,9 @@ def __init__(
     ) -> None:
         super().__init__()
         conn = self.get_connection(hive_cli_conn_id)
+
+        print("conn is from __init__", conn)
```

Comment:
Do we need this print?
This is going to be printed in the logs everything init is invoked when parsing the dag

Reply:
No, my bad, had added during debugging, forgot to remove it!